Modules/Components/itkImageRegistrationMethodv4/test/selxRegistrationItkv4Test.cxx

```diff
@@ -132,15 +132,15 @@ class RegistrationItkv4Test : public ::testing::Test
   typedef itk::TransformFileWriterTemplate<double> TransformWriterType;
   typedef itk::TransformFileReaderTemplate<double> TransformReaderType;
   
-  Logger::Pointer logger;
+  std::shared_ptr<Logger> logger;
```

Comment:
We should discuss this style of shared pointers to itk objects. I have not used this before myself.